src/main/java/ay2021s1_cs2103_w16_3/finesse/model/transaction/Date.java

```diff
@@ -21,6 +21,7 @@
     public static final DateTimeFormatter VALIDATION_FORMAT = DateTimeFormatter.ofPattern("dd/MM/uuuu")
             .withResolverStyle(ResolverStyle.STRICT);
     private static final LocalDate EPOCH = LocalDate.parse(EPOCH_STRING, VALIDATION_FORMAT);
+    private static final LocalDate CURRENT_DATE = LocalDate.now();
```

Comment:
I'm not sure if 'caching' the current date in a static variable is a good idea, as it may give unexpected results on legitimate behavior (e.g. when near midnight).

It might also make #246 harder to resolve.